index.html

```diff
@@ -240,15 +240,29 @@ <h1>Biometrics</h1>
   <div class="content">
     <div class="row">
     <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3>
-    <p> All cameras in Chicago's network may have still shots captured from them to use for identification purposes. </p>
+    <p> CPD has released very little information as to how they use FRT, although it seems that most of the infrastructure and personnel involved with FRT are located in the Crime Prevention and Information Center (CPIC), located in the Public Safety Headquarters. Still images from <a href="#cameras">cameras</a> in Chicago's camera network can be analyzed with FRT. </p>
     <h3> <span class="glyphicon glyphicon-piggy-bank"></span> Costs</h3>
-    <p><a href="https://www.muckrock.com/foi/chicago-169/mobile-biometric-technologies-chicago-police-department-20268/">FOIA documents</a> indicate that at least $63,000 have been spent on mobile fingerprint scanners. <a href="http://arstechnica.com/tech-policy/2014/06/first-chicago-robber-caught-via-facial-recognition-gets-22-years/">Other reporting</a> indicates upwards of $5.4 million has been spend on <a href="http://au.nec.com/en_AU/solutions/security-and-public-safety/biometrics/neoface-facial-recognition-overview.html">facial recognition technology</a>, enabling frames captured from a city camera to be compared with an existing database. </p>
+    <p><a href="https://www.muckrock.com/foi/chicago-169/mobile-biometric-technologies-chicago-police-department-20268/">FOIA documents</a> indicate that CPD spent $63,000 on mobile fingerprint scanners in 2013. It is unclear whether CPD has continued to purchase fingerprint scanning technology since then, although they have received <a href="https://www.documentcloud.org/documents/6191576-CPD-Integra-ID-MBIS-Upgrade-Proposal-Revised.html">proposals</a> for fingerprint scanners from NEC Corporation, which would have cost up to $5 million.</p>
+    <p>Documents obtained via FOIA by LPL reveal that in 2013, CPD paid $2.3 million to upgrade its <a href="https://www.documentcloud.org/documents/6174796-Facial-Recognition-Response/annotations/577372.html">camera network</a> and <a href="https://www.documentcloud.org/documents/6174796-Facial-Recognition-Response/annotations/577369.html">obtain FRT software</a> from <a href="http://www.dataworksplus.com/">Dataworks Plus</a>, funded via a grant from the <a href="https://www.muckrock.com/foi/chicago-169/facial-recognition-systems-chicago-police-department-35590/#file-153896">TSA</a>. These documents also indicate that CPD continued to pay around <a href="https://www.documentcloud.org/documents/6174796-Facial-Recognition-Response/annotations/577374.html">$100k a year</a> for FRT software until at least 2016. It is unclear how CPD has been using FRT since then. </p>
```

Comment:
Dataworks plus doesn't really do FRT. They tie the back-end data (mug shots) to algorithms sold by places like NEC. It's middle management software and this is a common misconception.

Reply:
I'll say "an FRT system" and then add stuff about the algorithms later on